src/index.ts

```diff
@@ -21,4 +21,4 @@ export * from './thread_manager';
 export * from './token_manager';
 export * from './types';
 export * from './channel_manager';
-export { isOwnUser, chatCodes, logChatPromiseExecution, formatMessage } from './utils';
+export { isOwnUser, chatCodes, logChatPromiseExecution, formatMessage, promoteChannel } from './utils';
```

Comment:
At some point, we should probably also export our other utils that might be useful for this; I was under the impression that all of them are exported (or at least the heavily tested ones).

Thoughts on this ? I can open a PR for it later